### PR TITLE
Refactor/update scss styling

### PR DESCRIPTION
**Pull request by jdxsmith**
#### What’s this PR do?
- Updates user input forms to display a message after posting
- Implemented SASS principles (mixins, variables, extends) to clean up `scss` file

#### Where should the reviewer start?
- In `_variables.scss` to see which variables/mixins are being created
- in `styles.scss` to see where the variables are being imported
- In `scripts.js' to see the functions added in order to display a message after posting data

#### How should this be manually tested?
- By running `npm start` in the terminal and entering data in any of the three forms

#### Any background context you want to provide?
- There is still some SASS refactoring to do (color functions)

#### What are the relevant tickets?
- #20 
- #21

### CODE REVIEW

**Comment by jaypeasee** [comment]
Great implementation of SASS principles to dry up our style sheets. I also like the ux of the form now too with the SetTimeout for display.

File: src/scripts.js [hunk 1]
```diff
@@ -97,12 +94,16 @@ function handleMetricSubmits(event) {
...
     .catch(error => console.log(error.message))
   }
 
-  function handleHydrationSuccess(message) {
+  function handlePostSuccess(className) {
+    let form = document.querySelector(`${className}`)
     loadMonitorData()
+    displaySuccessMessage(form)
+    form.reset();
+    setTimeout(function() { form.lastChild.innerHTML = '' }, 5000);
```
Comment (Codo-Baggins) [inline src/scripts.js:106]: Glad we made this dynamic, may want to handle other functions in a similar manner. This works beautifully too.  Good job here with the timeout.

File: src/scripts.js [hunk 2]
```diff
@@ -524,3 +525,9 @@ function makeFriendChallengeHTML(id, activityInfo, userStorage, method) {
 function makeStepStreakHTML(id, activityInfo, userStorage, method) {
   return method.map(streakData => `<li class="historical-list-listItem">${streakData}!</li>`).join('');
 }
+
+function displaySuccessMessage(form) {
```
Comment (Codo-Baggins) [inline src/scripts.js:529]: Great idea to give the user some confirmation that their data was posted.  We may want to add some stylings to the message to make it more evident to the user though.  The text is currently small and is almost buried in the form at the moment.

**Review by Codo-Baggins — APPROVED** [review]

**Comment by Codo-Baggins** [comment]
I went ahead and added a color function to change the side bar colors. They weren't very pretty before and I felt as though that changed belonged with this merge.

**Pull request merged**